.gitmodules

```diff
@@ -2,19 +2,7 @@
 	path = envoy
 	url = https://github.com/ipuustin/envoy.git
 	branch = openssl-container
-[submodule "QAT_Engine"]
```

Comment:
Good catch! I had forgotten `.gitmodules`.

Reply:
I used `git rm <submodule>` in ba8774e and it updated .gitmodules too, FYI